scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    cmdline="${cmdline} net.ifnames=0"
```

Comment:
> PR #380 was very helpfull

glad to hear :+1: I like it a lot myself :wink:

I'm actually still abusing it (with some jumping through hoops... oh well) to hook into the installer after fdisk is done, so I can set up an additional partition without the need for a separate reboot...